internal/testing/runchecks.sh

```diff
@@ -53,6 +80,10 @@ fi
   echo "FAIL: dependencies changed; compare listdeps.sh output with alldeps" && result=1
 }
 
+# Install wire; Moved here from the "install" step because we don't need to
```

Comment:
Nit: I don't think you need the comment, it's of historical interest only.

Reply:
Good point, removed.

Reply:
Oops, I forgot to push the commit that removes this comment, sorry. Will do in a followup